### PR TITLE
Add mkdir as part of .NET core install instructions

### PR DESCRIPTION
**Pull request by jamesmontemagno**
From issue: https://github.com/dotnet/cli/issues/3988

Else the rest will fail.

Also add in that you have to have Homebrew installed.

### CODE REVIEW

**Comment by bryanmacfarlane** [comment]
hmmm.  Didn't need to do.

Do these instructions also need updating?

https://www.microsoft.com/net/core#macos

**Comment by jamesmontemagno** [comment]
@bryanmacfarlane I would recommend adding it in. Essentially if you don't have a /usr/local/lib/ then it will fail to link. This causes issues with the vsts-agent. 

On a fresh machine it does not have it.  

I have now tested on 3 machines and 2 of the 3 times it has failed. Others have also ran into this issue.  As you can see here: https://github.com/dotnet/cli/issues/3988

**Comment by bryanmacfarlane** [comment]
OK - I had it on my fresh machine.  hmmm

Put commend above

# ensure directory exists

File: docs/start/envosx.md [hunk 1]
```diff
@@ -12,9 +12,12 @@ Net core requires update OpenSSL - [issue 110](https://github.com/Microsoft/vsts
 
 [From Net Core Instructions](https://www.microsoft.com/net/core#macos)
 
+In order to use .NET Core, we first need the latest version of OpenSSL. The easiest way to get this is from [Homebrew](http://brew.sh). After installing brew, do the following:
+
 ```bash
 brew update
 brew install openssl
+mkdir -p /usr/local/lib/
```
Comment (bryanmacfarlane) [inline docs/start/envosx.md]: put a blank line then comment 
\# ensure exists on fresh machine

Comment (jamesmontemagno) [inline docs/start/envosx.md]: How is that.

**Pull request merged**